### ISSUE TITLE
Type annotations / type hints / stub files for mypy

### ISSUE THREAD

**Issue by MartinThoma** [gangverk/flask-swagger#60]
Are there plans to add type annotations or stub files for mypy?

I've written an [article about type annotations](https://medium.com/analytics-vidhya/type-annotations-in-python-3-8-3b401384403d) if you need one :-)

**Comment by atlithorn** [comment]
Hey Martin! Thanks for the link, it's a fine read ;)

I think type annotations are sublime. 
I will be totally honest with you that there aren't all that many plans for flask-swagger.
I wrote this a while back to help enforce documentation of a project I was running and it has served us well.
Unfortunately my responsibilities have moved over to the managerial side of software development and I don't have the time to dedicate to moving this project forward. 

Having said that, dealing with PRs gives me much joy so please feel free to dive in if you are interested in moving us in that direction.

**Comment by MartinThoma** [comment]
Thank you! Good to know that flask-swagger is in maintenance mode.

Do you know [Flask-Restx](https://pypi.org/project/flask-restx/)? How does it compare to flask-swagger?

**Comment by atlithorn** [comment]
I don't know it but just by looking at their website it appears to be a framework. More similar to [Flask-RESTful](https://flask-restful.readthedocs.io/en/latest/) than flask-swagger?

If it uses MethodViews like Flask-RESTful then it's very likely it will work fine with flask-swagger.

**Comment by MartinThoma** [comment]
Yes, Flask-Restx is a fork of Flask-restful. I think they lost the access to the PyPI package (?)
